pkg/controllers/user/helm/common/common.go

```diff
@@ -207,7 +243,13 @@ func GenerateAnswerSetValues(app *v3.App, tempDir *HelmPath, extraArgs map[strin
 }
 
 func DeleteCharts(tempDirs *HelmPath, port string, obj *v3.App) error {
-	cmd := exec.Command(helmName, "delete", "--purge", obj.Name)
+	var cmd *exec.Cmd
+	if obj.Status.HelmVersion == HELMNAMEV2 || obj.Status.HelmVersion == "" {
```

Comment:
Feel like since we use this check so many times we could put into helper method: `isHelm3`, `isHelm2`